client/src/utils/tracks.js

```diff
@@ -72,6 +73,30 @@ export default class Tracks {
     })
   }
 
+  createOptions = async (added) => {
+    const prof = await getCurrentUserProfile()
+    const { data } = await getPlaylists(prof.data.id, added)
+    const options = this.convertToOptionsArray(data)
+
+    const filterOptions = options.filter((item) => {
+      if (item.value.spotifyId !== item.value.playlist.owner.id) {
```

Comment:
```
If (cond)
  return false
else
  return true
  ```
  Can generally be refactored to 
```
return !cond
```
I think this can become:
```
const filterOptions = options.filter((item) => item.value.spotifyId !== item.value.playlist.owner.id)
```